Wikipedia/UI-V5/WMFArticleListCollectionViewController.m

```diff
@@ -130,6 +143,21 @@ - (void)scrollToArticleIfOffscreen:(MWKArticle*)article animated:(BOOL)animated
 
 #pragma mark - UIViewController
 
+- (void)loadView {
```

Comment:
see my other comment about this not being a `UICollectionViewController`, which AFAIK doesn't provide a way to create the `UICollectionView`—only allows custom layouts